broadcastclient/broadcastclient.go

```diff
@@ -272,16 +290,32 @@ func (bc *BroadcastClient) startBackgroundReader(earlyFrameData io.Reader) {
 
 				if res.Version == 1 {
 					if len(res.Messages) > 0 {
-						var messages []arbstate.MessageWithMetadata
 						for _, message := range res.Messages {
 							if message == nil {
 								log.Warn("ignoring nil feed message")
 								continue
 							}
+
+							if message.Message.Message.Header.RequestId == nil && bc.config.RequireSignature {
+								// Missing request id prevents signature from being checked
+								log.Error("ignoring feed message with missing request id", "sequence_number", message.SequenceNumber)
```

Comment:
I believe sequencer messages don't have a request id, so we'll need to handle them